### PR TITLE
Providing line number of first out-of-order key.

### PR DESCRIPTION
**Pull request by rozenbom**

### CODE REVIEW

**Comment by eprykhodko** [comment]
@rozenbom I think the error message should be changed to `Sort keys. 'a'  should be before 'b'`
I'm also considering possibility of printing `the correct order should be ['a', 'b', 'c']` if there are fewer than, say, 5 keys and the  `... 'a'  should be before 'b' ` if there are more

@yevhen-m  what do you think?

**Comment by yevhen-m** [comment]
@eprykhodko I don't like this idea about "correct order".

File: flake8_sorted_keys/checker.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ class SortedKeysChecker(object):
     name = 'flake8-sorted-keys'
     version = __version__
 
-    unsorted_message = 'S001 Sort keys'
+    unsorted_message = 'S001 Sort keys. \'{0}\' should be before \'{1}\'.'
```
Comment (eprykhodko) [inline flake8_sorted_keys/checker.py]: "S001 Sort keys. '{0}' should be before '{1}'."

File: tests/test_checker.py [hunk 2]
```diff
@@ -29,18 +29,19 @@ def test_unsorted_keys(self):
         self.assertEqual(len(lint_errors), 1)
         first_error = lint_errors[0]
         line, offset, msg, _ = first_error
-        self.assertIn('S001', msg)
+        self.assertEqual(line, 4)
+        self.assertEqual(msg, 'S001 Sort keys. \'b\' should be before \'c\'.')
```
Comment (yevhen-m) [inline tests/test_checker.py]: Same here please.

File: flake8_sorted_keys/checker.py [hunk 3]
```diff
@@ -7,7 +7,7 @@ class SortedKeysChecker(object):
     name = 'flake8-sorted-keys'
     version = __version__
 
-    unsorted_message = 'S001 Sort keys'
+    unsorted_message = "S001 Sort keys. '{0}' should be before '{1}'."
```
Comment (yevhen-m) [inline flake8_sorted_keys/checker.py:10]: Should we change `unsorted_message` to `message`?

Comment (yevhen-m) [reply flake8_sorted_keys/checker.py:10]: @eprykhodko, what about a smiley after `Sort keys`?

Comment (rozenbom) [reply flake8_sorted_keys/checker.py:10]: :(

**Review by eprykhodko — APPROVED** [review]
legit

**Pull request merged**